python/parse_sosreport.py

```diff
@@ -645,7 +645,7 @@ def find_directory(self, directory):
         return result
 
     @staticmethod
```

Comment:
It doesn't work. You have to remove @staticmethod in that case.

The method is not used. It should be removed instead.